components/style/values/computed/mod.rs

```diff
@@ -41,6 +44,13 @@ pub trait ToComputedValue {
 
     #[inline]
     fn to_computed_value(&self, _context: &Context) -> Self::ComputedValue;
+
+    #[inline]
+    /// Convert a computed value to specified value form.
+    ///
+    /// This will be used for recascadeing during animation.
+    /// Such to_specified_valued values should recompute to the same value.
+    fn to_specified_value(computed: &Self::ComputedValue, context: &UncomputeContext) -> Self;
```

Comment:
I like your suggestion of calling this `from_computed_value`.  Whether it makes sense to rename the trait to something like `ToFromComputedValue` or `ComputedValueConversion` I'm not sure.